nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -995,6 +995,11 @@ impl<'a> Planner<'a> {
                         "kind" => ?zone.zone_type.kind(),
                         "image_source" => %image_source,
                     );
+                    self.blueprint.comment(format!(
+                        "upgrade {:?} zone {} in-place",
```

Comment:
```suggestion
                        "updating {:?} zone {} in-place",
```
I think we use "update" instead of "upgrade" in most cases, right? I like to keep it consistent to aid `grep`.

Reply:
Good call.  Fixed.